kijiji_repost_headless/kijiji_api.py

```diff
@@ -115,6 +117,13 @@ def login(self, username, password):
         }
         api_url = 'https://www.kijiji.ca/anvil/api'  # API endpoint
         resp = self.session.post(api_url, json=payload)
+        """
+        
+        parent_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        ssid_path = os.path.join(parent_dir, "ssid.txt")
```

Comment:
Is it possible to load string `ssid` as a parameter? Similar to what we currently do with the username / password?

Reply:
It's possible to treat SSID like username and password by asking for input during the construction of the YAML files, but I thought it was messy. The SSID is 340 characters and must be fetched and pasted, not remembered and typed. Asking for it each time a new YAML file is created is not user-friendly. By contrast, pasting it exactly once into a blank text file during setup is simpler.

The code for pulling the SSID from that text file can easily be moved (into __main__.py, for instance) if you think that it's out of place in kijiji_api.py. Depends what you're looking to achieve.